workspace/teampipeline/generate_report.py

```diff
@@ -175,6 +179,8 @@ def get_slack_username(github_username):
 
 
 def get_status_and_summary(card):  # pragma: no cover
+    if not card["fieldValues"]["nodes"][-1]:
```

Comment:
There are cards on the board with no status, where `'nodes': [{}, {}]`

Reply:
Thank you for the fix!